mail/common/src/test/java/com/fsck/k9/mail/internet/MimeParameterDecoderTest.kt

```diff
@@ -33,9 +34,10 @@ class MimeParameterDecoderTest {
         )
 
         assertThat(mimeValue.value).isEqualTo("message/external-body")
-        assertThat(mimeValue.parameters).containsExactlyEntries(
-            "access-type" to "URL",
+        // FIXME does the order matter? If so, we need to fix MimeParameterDecoder, as it changes the order
```

Comment:
Does the order matter? If so, we need to fix MimeParameterDecoder, as it changes the order and I had to move `"access-type" to "URL",` to second position for the test to succeed